### ISSUE TITLE
loss function vs epochs for mictotubules data

### ISSUE THREAD

**Issue by mahirahzaini** [qc17-THU/DL-SR#7]
Hi, I'm just wondering if you have any data of training loss vs the epochs for microtubules data?

**Comment by qc17-THU** [comment]
The typical training and validation loss of microtubules are similar to those of CCP data which were provided in the Supplementary Figure 13 in our NM manuscript.
